src/text.rs

```diff
@@ -212,37 +218,70 @@ impl<'a> Spans<'a> {
     /// assert_eq!(7, spans.width());
     /// ```
     pub fn width(&self) -> usize {
-        self.0.iter().map(Span::width).sum()
+        self.content.iter().map(Span::width).sum()
+    }
+
+    /// Changes the alignment of this string.
+    ///
+    /// ## Examples
+    ///
+    /// ```rust
+    /// # use ratatui::layout::Alignment;
+    /// # use ratatui::text::{Span, Spans};
+    /// # use ratatui::style::{Color, Style};
+    /// let spans = Spans::from(vec![
+    ///     Span::styled("My", Style::default().fg(Color::Yellow)),
+    ///     Span::raw(" text"),
+    /// ]).aligned(Alignment::Right);
+    /// assert_eq!(Some(Alignment::Right), spans.alignment);
+    /// ```
+    pub fn aligned(self, alignment: Alignment) -> Self {
```

Comment:
nit: call this `fn align`  or `fn alignment` to match other widget setters.

Reply:
This is resolved in the current version.